build/yaml/crd/vpc/crd.nsx.vmware.com_ipaddressallocations.yaml

```diff
@@ -16,65 +17,63 @@ spec:
         properties:
           apiVersion:
-            description: |-
-              APIVersion defines the versioned schema of this representation of an object.
-              Servers should convert recognized schemas to the latest internal value, and
-              may reject unrecognized values.
-              More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
             type: string
           kind:
-            description: |-
-              Kind is a string value representing the REST resource this object represents.
-              Servers may infer this from the endpoint the client submits requests to.
-              Cannot be updated.
-              In CamelCase.
-              More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
             type: string
           metadata:
             type: object
           spec:
             description: IPAddressAllocationSpec defines the desired state of IPAddressAllocation.
             properties:
-              allocation_size:
-                description: AllocationSize specifies the size of IP CIDR to be allocated.
+              allocationSize:
+                description: AllocationSize specifies the size of allocation_ips to
+                  be allocated.
                 type: integer
-              ip_address_block_visibility:
+              ipAddressBlockVisibility:
                 default: Private
                 description: IPAddressBlockVisibility specifies the visibility of
-                  the IPBlocks to allocate IP addresses. Can be External or Private.
+                  the IPBlocks to allocate IP addresses. Can be External, Private
+                  or PrivateTGW.
                 enum:
                 - External
                 - Private
+                - PrivateTGW
                 type: string
             type: object
           status:
             description: IPAddressAllocationStatus defines the observed state of IPAddressAllocation.
             properties:
-              CIDR:
-                description: CIDR is the allocated CIDR
+              allocationIPs:
+                description: allocation_ips is the allocated IP addresses
```

Comment:
allocation_ips -> allocationIPs

Reply:
I think here it should be `allocation_ips` -> `AllocationIPs`, keeping the same with `AllocationSize specifies ...`
Because previously in the description we used the UpperCamelCase, e.g. https://github.com/vmware-tanzu/nsx-operator/blob/4e4decd56d10720d662eb4e23c78e2eeec65a363/build/yaml/crd/vpc/crd.nsx.vmware.com_securitypolicies.yaml#L442
And in K8s, both UpperCamelCase and lowerCamelCase have the samples, e.g.
1. UpperCamelCase:
```
# kubectl explain pod.spec
...
   hostAliases  <[]Object>
     HostAliases is an optional list of hosts and IPs that will be injected into
     the pod's hosts file if specified. This is only valid for non-hostNetwork
     pods.
...
   nodeName     <string>
     NodeName is a request to schedule this pod onto a specific node. If it is
     non-empty, the scheduler simply schedules this pod onto that node, assuming
     that it fits resource requirements.
...
```
2. LowerCamelCase:
```
# kubectl explain pod.status
...
   podIP        <string>
     podIP address allocated to the pod. Routable at least within the cluster.
     Empty if not yet allocated.
...
```
So I think we just need to keep it similar with the previous style.